packages/convertkit/src/forms/subscribe/index.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react'
+import * as Yup from 'yup'
+import axios from 'axios'
+import queryString from 'query-string'
+import {useFormik} from 'formik'
+import {useRouter} from 'next/router'
+import {CK_SUBSCRIBER_KEY} from '@skillrecordings/config'
+import {Input, Button} from '@skillrecordings/react/dist/components'
+
+type SubscribeFormOptions = {
+  /**
+   * The content of submit button.
+   * @type string
+   */
+  actionLabel?: string
+  /**
+   * The URL to redirect to after succesfully submitting a form. Default is `/confirm`.
+   * Submitted `email_address` will be automatically passed as URL param.
+   * Providing `false` value will skip the redirect entirely.
+   * @type string | false
+   */
+  onSuccessRedirectUrl?: string | false
```

Comment:
i think this should be a callback and not explicitly redirect to a url, so just `onSuccess` and let us (the user) choose what action to take

Reply:
great idea 👍